lib/elixir/lib/kernel.ex

```diff
@@ -1477,6 +1477,7 @@ defmodule Kernel do
       false
 
   """
+  def "" =~ "", do: true
```

Comment:
I think it should actually be `def _ =~ "", do: true` as `"foo" =~ ""` also fails today.